docs/modules/ROOT/pages/references/release_notes.adoc

```diff
@@ -2,6 +2,63 @@
++
+For more information, see https://docs.redhat.com/en/documentation/openshift_container_platform/4.20/html-single/authentication_and_authorization/index#external-auth[the official documentation].
+
+External Secrets Operator becomes Generally Available::
+The External Secrets Operator is a cluster service that provides lifecycle management for secrets fetched from external secret management systems (such as AWS Secrets Manager, HashiCorp Vault, and Azure Key Vault).
+The operator provisions, fetches and refreshes the secret within the cluster, ensuring a secure and efficient secrets flow without direct application involvement.
++
+For more information, see https://docs.redhat.com/en/documentation/openshift_container_platform/4.20/html/security_and_compliance/external-secrets-operator-for-red-hat-openshift[the official documentation].
+
+OpenShift AI gains new capabilities::
+OpenShift Container Platform 4.20 brings several new capabilities to OpenShift AI that improve the scalability of AI workflows, such as simplified deployments of distributed AI workloads (leveraging the LeaderWorkerSet resource), and improved load balancing for distributed inference with llm-d (leveraging the Kubernetes Gateway API Inference Extensions).
++
+For more information, see https://gateway-api-inference-extension.sigs.k8s.io/[the official documentation].
+
+Multiple network interface controllers on vSphere clusters becomes Generally Available::
+For clusters on VMware vSphere, it has been possible since OpenShift 4.18 to set up the cluster with multiple network interface controllers for one node.
+This feature is now becoming Generally Available.
++
+For more information, see https://docs.redhat.com/en/documentation/openshift_container_platform/4.20/html-single/installing_on_vmware_vsphere/index#installation-vsphere-multiple-nics_installing-vsphere-installer-provisioned-customizations[the official documentation].
+
+Linux User Namespace Support becomes Generally Available::
+Support for deploying pods into Linux user namespaces is now generally available and enabled by default.
+This feature improves isolation, mitigating security vulnerabilities that one compromised container can pose to other pods and the node itself.
++
+This change includes two new security context constraints, `restricted-v3` and `nested-container`, which are designed for use with user namespaces.
++
+For further information, see https://docs.redhat.com/en/documentation/openshift_container_platform/4.20/html-single/nodes/index#nodes-pods-user-namespaces[the official documentation].
+
+Docker v2 registries become Deprecated::
+Support for Docker v2 registries will be removed in a future release.
+At that point, all mirroring operations will require a registry that supports the OCI specification.
+
+Red Hat Marketplace becomes Deprecated::
+Customers using software from the Marketplace should reach out to the software vendor to find out how to migrate away from the Marketplace Operator.
```

Comment:
Maybe a slightly better explanation here what the Marketplace is for customers reading our docs?